opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinionsRadarDialog.tsx

```diff
@@ -81,7 +83,21 @@ StixCoreObjectOpinionsRadarDialogProps
     myOpinionValue ?? Math.round(opinionOptions.length / 2),
   );
   const [open, setOpen] = useState(false);
-  const handleOpen = () => setOpen(true);
+  const handleOpen = () => {
+    if (opinionOptions.length > 0) {
+      setOpen(true);
+    } else {
+      MESSAGING$.notifyError(
+        <span>
+          {t('There are no opinions. You can create some ')}{' '}
+          <Link to="/dashboard/settings/vocabularies/fields/opinion_ov">
+            {t('in this page')}
+          </Link>
```

Comment:
It seems we miss translations here

Reply:
And are we sure we want to add a link to an admin page when majority of people won't have access to it?

Reply:
mmm yes you're right, I will rather add a warning message with no redirection link.

Reply:
fixed